src/views/SpaceSettings.vue

```diff
@@ -190,32 +190,34 @@ function setAvatarUrl(url) {
   if (typeof url === 'string') form.value.avatar = url;
 }
 
-onMounted(async () => {
-  try {
-    const uri = await getSpaceUri(props.spaceId);
-    console.log('URI', uri);
-    currentTextRecord.value = uri;
-    const space = clone(props.space);
-    if (!space) return;
-    delete space.id;
-    delete space._activeProposals;
-    space.strategies = space.strategies || [];
-    space.plugins = space.plugins || {};
-    space.validation = space.validation || basicValidation;
-    space.filters = space.filters || {};
-    currentSettings.value = clone(space);
-    form.value = space;
-  } catch (e) {
-    console.log(e);
-  }
-  if (props.from) {
-    const fromClone = clone(props.spaceFrom);
-    fromClone.validation = fromClone.validation || basicValidation;
-    delete fromClone.id;
-    delete fromClone._activeProposals;
-    form.value = fromClone;
+watchEffect(async () => {
+  if (!props.spaceLoading) {
+    try {
+      const uri = await getSpaceUri(props.spaceId);
+      console.log('URI', uri);
```

Comment:
Do we need this console?

Reply:
@bonustrack do you want to keep this? Why is it needed?

Reply:
I use it from time to time to debug space, but you can remove it